src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -49,14 +49,19 @@ export const AuctionArtworksRail: React.FC<AuctionArtworksRailProps> = ({
             </Text>
           )}
         </Box>
-
-        <Box height={AUCTION_ARTWORKS_RAIL_HEIGHT}>
-          {isEnteredView ? (
-            <AuctionArtworksRailArtworksQueryRenderer id={sale.internalID} />
-          ) : (
-            <AuctionArtworksRailPlaceholder />
-          )}
-        </Box>
+        {/* Only showing Artwork Images for Current and Past Auctions */}
+        {(!sale.isLiveOpen && sale.status == "open") ||
```

Comment:
In JS we'll always want to be sure to use [strict equality](https://stackoverflow.com/questions/523643/difference-between-and-in-javascript)

Reply:
Removed this.